cg/meta/compress/compress.py

```diff
@@ -33,6 +40,17 @@ def set_dry_run(self, dry_run: bool):
         if self.crunchy_api.dry_run is False:
             self.crunchy_api.set_dry_run(dry_run)
 
+    def get_flow_cell_name(self, fastq_path: Path) -> str:
```

Comment:
Did you add a unit test for this new function?

Reply:
done